acceptance/Gemfile

```diff
@@ -9,5 +9,5 @@ gem "inspec"
 # puts in a box_url for bento when a vagrant box in atlas is specified
 gem "kitchen-vagrant", github: "test-kitchen/kitchen-vagrant"
 gem "windows_chef_zero"
-gem "winrm-fs", "~> 0.4.0"
+gem "winrm-fs"
```

Comment:
I forget: was this just here because we wanted to pin the version? Does kitchen bring this in on its own?